index.bs

```diff
@@ -556,8 +563,34 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
 1. [=list/Remove=] all entries in |cache| where the entry's [=attribution source/expiry time=] is less than the current time.
 1. If the [=list/size=] of |cache| is greater than or equal to the user agent's
      [=max source cache size=], return.
+1. If |source|'s [=attribution source/attribution mode=] is "<code>falsely</code>", then:
+    1. Assert: |source|'s [=attribution source/source type=] is "<code>event</code>".
+    1. Let |fakeReport| be a new [=attribution report=] with the items:
+        : [=attribution report/event id=]
+        :: |source|'s [=attribution source/event id=]
+        : [=attribution report/trigger data=]
+        :: A random value between 0 (inclusive) and the user agent's
+            [=max event source trigger data value=] (exclusive)
+        : [=attribution report/reporting endpoint=]
+        :: |source|'s [=attribution source/reporting endpoint=]
+        : [=attribution report/attribution destination=]
+        :: |source|'s [=attribution source/attribution destination=]
+        : [=attribution report/reporting time=]
+        :: The result of running [=obtain a report delivery time=] with |source| and |source|'s
+            [=attribution source/source time=]
+        : [=attribution report/trigger priority=]
+        :: 0
+        : [=attribution report/trigger time=]
+        :: |source|'s [=attribution source/source time=]
+        : [=attribution report/source identifier=]
+        :: |source|'s [=attribution source/source identifier=]
+    1. Add |fakeReport| to the [=attribution report cache=].
+    1. Return.
 1. [=set/Append=] |source| to |cache|.
 
+Issue: Should fake reports respect the user agent's [=max report cache size=] and/or
```

Comment:
Good call out! I don't see why this should not respect the max report cache size.

As for max reports per attribution destination, that seems a bit more nuanced and leaving as an issue sgtm.